content/overview/installation/_index.en.md

```diff
@@ -70,99 +49,23 @@ choco install wash
-mkdir -p wasmcloud
-tar -xvf aarch64-linux.tar.gz -C wasmcloud
-```
-
-{{% /tab %}}
-{{% tab "Intel Mac" %}}
-
-```bash
-wget https://github.com/wasmCloud/wasmcloud-otp/releases/download/v0.55.1/x86_64-macos.tar.gz
-mkdir -p wasmcloud
-# This command makes it so the MacOS Gatekeeper will not quarantine parts of the host when you run it:
-sudo xattr -d com.apple.quarantine x86_64-macos.tar.gz
-tar -xvf x86_64-macos.tar.gz -C wasmcloud
-```
-
-{{% /tab %}}
-{{% tab "M1 Mac" %}}
-
-```bash
-wget https://github.com/wasmCloud/wasmcloud-otp/releases/download/v0.55.1/aarch64-macos.tar.gz
-mkdir -p wasmcloud
-# This command makes it so the MacOS Gatekeeper will not quarantine parts of the host when you run it:
-sudo xattr -d com.apple.quarantine aarch64-macos.tar.gz
-tar -xvf aarch64-macos.tar.gz -C wasmcloud
-```
-
-{{% /tab %}}
-{{% tab "Windows" %}}
-
-```powershell
-wget https://github.com/wasmCloud/wasmcloud-otp/releases/download/v0.55.1/x86_64-windows.tar.gz
-mkdir wasmcloud
-tar -xvf x86_64-windows.tar.gz -C wasmcloud
-```
-
 {{% /tab %}}
 {{% /tabs %}}
 
-After extracting from the tar file, the host is fully installed, and the tar file can be deleted. If you haven't already, run NATS with Jetstream following the instructions [above](#install-nats-server).
-
-There are a variety of ways to run the host that are described in more detail in [Running the Host](/reference/host-runtime/running). For now, go ahead and `cd wasmcloud` to get into the correct directory and then run:
-```bash
-bin/wasmcloud_host foreground
-```
-This will start the host with attached logs and can be exited at any time by doing `ctrl-c`. Now, you're ready to proceed onto [Getting started](/overview/getting-started/).
-
-#### Install from source
-The [wasmCloud OTP host runtime](https://github.com/wasmCloud/wasmcloud-otp) and [wash](https://github.com/wasmcloud/wash) repositories are open source on GitHub. Both of these repositories can be cloned and built locally directly. Consult the README file in the respective repository for prerequisites and instructions for compiling and running.
-
-### Stopping the wasmCloud Host Runtime
+Once `wash` is installed, simply run `wash up`. This will download and start NATS and wasmCloud for you, printing logs to the terminal. You can exit at any time by pressing `ctrl-c`. Now, you're ready to proceed onto [Getting started](/overview/getting-started/).
 
-There are a number of ways you can stop the runtime, and the recommendation varies with how you started the host. For details, see the [safe shutdown](/reference/host-runtime/safeshutdown) section of the reference guide.
+> For convenience we provide a Docker Compose file if you prefer to use Docker instead of `wash up` to launch NATS and wasmCloud. You can follow our [install with Docker](./install-with-docker/) instructions to do so, but it's still recommended to install `wash` even if you don't make use of the `up` feature.

```

Comment:
Doesn't need to be now, but this line got me thinking - could we do a `wash up --docker` command that starts all of the services as docker containers?

Reply:
I like that idea, we could simply embed a docker compose file and use `--docker` to shell out to the docker command line